app/views/internal/events/index.html.erb

```diff
@@ -1,23 +1,26 @@
-<div class="markdown events-index">
-  <% if @success %>
-    <div class="confirmation-panel-container">
-      <div class="govuk-panel govuk-panel--confirmation">
-        <h1 class="govuk-panel__title">
-          <%= @pending ? "Event submitted for review" : "Event published" %>
-        </h1>
-        <% if @readable_id %>
-          <div class="govuk-panel__body">
-            <%= @pending ? "Pending event" : "Published event" %><br>
-            <strong>
-              <%= link_to @readable_id,
-                          @pending ? internal_event_path(@readable_id) : event_path(@readable_id) %>
-            </strong>
-          </div>
-        <% end %>
-      </div>
+<% if @successful %>
+  <div class="confirmation-panel-container">
+    <div class="govuk-panel govuk-panel--confirmation">
+      <h1 class="govuk-panel__title">
+        <%= "Event submitted for review" if @pending %>
```

Comment:
Can be picked up later, but I think the text dependent on status could be cleaner if we moved the text to the translations file and had something like:

```
<h1 class="govuk-panel__title">
<%= t("some.key.#{@status}") %>
</h1>
```